web/client/components/import/style/StylePanel.jsx

```diff
@@ -138,6 +146,17 @@ class StylePanel extends React.Component {
         </Row>);
     };
 
+    renderDisableStyleCustomization = () => {
+        if (this.state.disableStyleCustomization) {
+            return (
+                <div className="alert alert-info mb-2 style-customisation-disabled-container">
+                    <Message msgId="shapefile.styleCustomizationDisabled"/>
+                </div>
+            );
+        }
+        return null;
+    }
```

Comment:
```suggestion
    renderDisableStyleCustomization = (hasCustomStyle) => {
            return hasCustomStyle ? (
                <div className="alert alert-info mb-2 style-customisation-disabled-container">
                    <Message msgId="shapefile.styleCustomizationDisabled"/>
                </div>
            );
        } : null;
    }
```

Reply:
Hey @MV88  How do I get the has `hasCustomStyle ` value.